build/scripts/scripts.fsproj

```diff
@@ -16,7 +16,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Buildalyzer" Version="3.2.0" />
-    <PackageReference Include="FSharp.Core" Version="4.7.2" />

```

Comment:
Hmm, I'm curious; how did you find this?

Reply:
This was the first warning produced by running `./build.sh`:

```
apm-agent-dotnet/build/scripts/scripts.fsproj : warning NU1504: Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: FSharp.Core 6.0.6, FSharp.Core 4.7.2.
```

As `FSharp.Core` is referenced by default, I simply removed the `PackageReference`.